src/style.css

```diff
@@ -0,0 +1,108 @@
+* {
```

Comment:
Margin and padding reset recommended to apply to specific tags. Using * for box-sizing: border-box is ok.